roles/openstack-setup/tasks/users.yml

```diff
@@ -15,6 +15,8 @@
                     login_password={{ secrets.admin_password }}
                     insecure={{ insecure|default(omit) }}
   with_items: "{{ keystone.services }}"
-  when: "{{item.name}}.enabled is undefined or {{item.name}}.enabled|bool"
+  when: 
+    - (item.name.enabled is undefined or item.name.enabled|bool)
```

Comment:
we don't explicitly enable some of the default services that need endpoints like keystone, keystonev3, and glance, so any of the services that aren't always present will either be disabled in envs, or defaults-2.0.yml